metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/parser/EnaObjectCollectorTest.java

```diff
@@ -0,0 +1,118 @@
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.parser;
+
+import org.apache.xmlbeans.XmlException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.util.ResourceUtils;
+import uk.ac.ebi.ampt2d.metadata.enaobject.EnaObjectCollector;
+import uk.ac.ebi.ampt2d.metadata.service.EnaDbService;
+import uk.ac.ebi.ena.sra.xml.AnalysisFileType;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.SQLXML;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.anyMap;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+@TestPropertySource("classpath:application.properties")
+public class EnaObjectCollectorTest {
+
+    @Mock
+    private NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Mock
+    private SQLXML sqlxml;
+
+    @Mock
+    private EnaDbService enaDbService;
+
+    @InjectMocks
+    private EnaObjectCollector enaObjectCollector;
+
+    @Before
+    public void initialization() throws Exception {
+        List<SQLXML> sqlxmlList = new ArrayList<>();
+        sqlxmlList.add(sqlxml);
+        when(sqlxml.getString()).thenReturn(getXmlFile("classpath:ERZ000011.xml"));
+        when(jdbcTemplate.queryForObject(anyString(), anyMap(), eq(Long.class))).thenReturn(Long.valueOf(5));
+        when(jdbcTemplate.queryForList(anyString(), anyMap(), eq(SQLXML.class))).thenReturn(sqlxmlList);
+        when(enaDbService.getEnaAnalysisXml(anyLong(), anyLong())).thenReturn(sqlxmlList);
+    }
+
+    private String getXmlFile(String fileName) throws IOException {
+        File file = ResourceUtils.getFile(fileName);
+        return new String(Files.readAllBytes(file.toPath()));
+    }
+
+    @Test
+    public void testGetEnaAnalysisFileTypeFromDb() {
+        List<AnalysisFileType> analysisFileTypeList = enaObjectCollector.getEnaAnalysisFileTypeFromDb(1, 2);
+        assertAnalysisFileTypeList(analysisFileTypeList);
+    }
+
+    private void assertAnalysisFileTypeList(List<AnalysisFileType> analysisFileList) {
+        assertEquals(analysisFileList.size(), 2);
```

Comment:
The expected value should go first, see [Javadoc](http://junit.sourceforge.net/javadoc/org/junit/Assert.html).